commands/generate.js

```diff
@@ -0,0 +1,111 @@
+'use babel';
+
+var fs = require('fs');
```

Comment:
You're using `babel`, so take advantage of ES2015, like using `const` instead of `var`.